ipatests/test_xmlrpc/test_automember_plugin.py

```diff
@@ -406,6 +498,79 @@ def test_delete_deps_for_rebuilding_hostgroups(self, host1, hostgroup1,
         automember_hostgroup.delete()
 
 
+class TestAutomemberModifyNegativeSameval(XMLRPC_test):
```

Comment:
A better name would be `TestAutomemberModifyNegative` (same val applies only to one of the tests)